s2a/src/test/resources/int_cert1_.cnf

```diff
@@ -0,0 +1,14 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+prompt = no
+
+[req_distinguished_name]
+O = o
```

Comment:
Every level has the same subject name? That just seems like we hate our future selves if we ever have to debug. Let's make them unique in some way.

Reply:
Done in https://github.com/grpc/grpc-java/pull/11583/commits/a48dc83c62433f27b886d406bf99b2fbb376b13a